### ISSUE TITLE
Report end time at the end of the test

### ISSUE THREAD

**Issue by damies13** [damies13/rfswarm#51]
**Describe the solution you'd like**
> And also test end time in the console?

![image](https://user-images.githubusercontent.com/15232871/71062113-72f9eb80-21b5-11ea-8138-2ca83d8a5393.png)

**Comment by damies13** [comment]
I thought I had done that, but when I checked the code, it only happens if you used one of the auto-run modes, I guess I need to implement this for manual run as well. I logged Issue #51 for this.

This is the code that detects the end of a test:
```
if base.run_end > 0 and base.run_end < int(time.time()) and base.total_robots < 1 and not base.posttest:
```
Unfortunately it is inside an if clause that is triggered by the -r auto-run command line switch

**Comment by damies13** [comment]
Fix Merged into Release v0.6.2
